go.mod

```diff
@@ -1,106 +1,38 @@
 module github.com/harmony-one/harmony
```

Comment:
why change go.mod again?

Reply:
current go.mo not work on my machine, I have to generate a new go.mo